_candidates_en/KimuraSotaro_1.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Sotaro Kimura"
+title: "Stream data processing 101"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: kimutansk
+github: kimutansk
+tags:
+  - Big Data / Fast Data
+---
+There's an increasing demand for stream data processing, which processes ever-occurring data in real-time, visualizes the results, or handles the resulting event. In this session, we will see some of the aspects of stream data processing, how they tackle the issues of batch processing, and it newly enables. Just in the last few years there have been a number of stream data processing platforms such as Spark Streaming, Flink, Apex, Gearpump, and Beam; so much so that it's unclear from the users' perspective where we are headed. By illustrating the history of stream data processing on the JVM, along with comparison of products that came out in the last few years, I hope to clarify the key points of the platforms that developers should know when building a stream data processing system.
```

Comment:
Line 2: Replace "see" with "explore"?
Line 2: Who is "they"; developers? Need to define.
Line 3: Should "and it newly enables" read "and what technology it newly enables"?
Line 4: There should be a verb between "processing platform" and "such as"; perhaps add "created"?
Line 4: Replace "a number" with "many"?
Line 5: Should JVM be written out or is this a common abbreviation?
Line 5: Replace "know" with "understand"?